metrics/github/client.py

```diff
@@ -52,6 +52,11 @@ def rest_query(self, path, **variables):
             data = response.json()
             if isinstance(data, list):
                 yield from data
+            # unlike the PRs/Issues/Commits GitHub API endpoints
+            # the codespaces endpoint returns a dict containing a
+            # count of codespaces, and a list thereof (what we want)
```

Comment:
I found this comment confusing because we don't fetch PRs and issues from the REST API and don't fetch commits at all. I think you want to say:

> Unlike the team repositories endpoint or the team members endpoint, which return arrays of the objects we're interested in, the codespaces endpoint returns an object. This object has a `codespaces` key, whose value is an array of the objects we're interested in.